lib/tasks/canonical_models/canonical_potions.json

```diff
@@ -1551,11 +1876,14 @@
   },
   {
     "attributes": {
-      "name": "Elixir of Resist Magic",
```

Comment:
Elixir of Resist Magic is unobtainable and Elixir of Resist Shock was not in the JSON data. This diff makes it look as if the former turned into the latter, when in fact the former was removed and the latter added.